app/src/main/java/com/quran/labs/androidquran/pageselect/PageSelectPresenter.kt

```diff
@@ -42,8 +43,10 @@ class PageSelectPresenter @Inject
         } else if (!downloadingSet.contains(it.key)){
           downloadingSet.add(it.key)
           val url = "$baseUrl/${it.key}.png"
+          val backupUrl = "$backupBaseUrl/${it.key}.png"
           compositeDisposable.add(
               imageUtil.downloadImage(url, previewImage)
+                  .onErrorResumeNext(imageUtil.downloadImage(backupUrl, previewImage))
```

Comment:
Elsewhere you are `replace`ing on error, here a separate variable. Maybe use a common convention?

Actually, I personally would prefer to have a single utility class/method for all `replace`ments :)

Reply:
jazakAllah khairan - done